packages/fuchsia_remote_debug_protocol/lib/src/dart/dart_vm.dart

```diff
@@ -129,23 +129,16 @@ class DartVm {
   ///
   /// This is not limited to Isolates running Flutter, but to any Isolate on the
   /// VM.
```

Comment:
Probably want to add a line here warning about avoiding the use of a non-exhaustive pattern.  I'm not sure if this is still the case, but if you don't add something ending in ".cmx", say you have a `example_app` and `example_app_test` it's possible that attempting to connect to just `example_app` would end up matching `example_app_test` and crash since none of the flutter driver extensions would be enabled.

Reply:
Updated comment.